src/Outputs/Alert.php

```diff
@@ -14,8 +14,25 @@ public function boot()
 
     public function output(Collection $detectedQueries, Response $response)
     {
-        if (stripos($response->headers->get('Content-Type'), 'text/html') !== 0 || $response->isRedirection()) {
-            return;
+
+        $contentType = $response->headers->get('Content-Type');
+        
+        $validateContentTypeHTML = true;
+
+        if (env('QUERY_DETECTOR_AJAX', false)) {
```

Comment:
You need to use `config(...)`, not `env(...)` because when config are cached `env()` will always return `null`.

Config reference:
https://laravel.com/docs/7.x/configuration#configuration-caching